[2025-09-24T09:09:34.908+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_d8 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:09:34.920+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_d8 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:09:34.921+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T09:09:34.937+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cleanse_data_d8> on 2025-09-24 08:00:00+00:00
[2025-09-24T09:09:34.956+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'cleanse_data_d8', 'scheduled__2025-09-24T08:00:00+00:00', '--job-id', '246', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmplw4yazd1']
[2025-09-24T09:09:34.956+0000] {standard_task_runner.py:60} INFO - Started process 1656 to run task
[2025-09-24T09:09:34.961+0000] {standard_task_runner.py:88} INFO - Job 246: Subtask cleanse_data_d8
[2025-09-24T09:09:35.033+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_d8 scheduled__2025-09-24T08:00:00+00:00 [running]> on host feaba2204362
[2025-09-24T09:09:35.122+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='cleanse_data_d8' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-24T08:00:00+00:00'
[2025-09-24T09:09:35.332+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:09:35.360+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=cleanse_data_d8, execution_date=20250924T080000, start_date=20250924T090934, end_date=20250924T090935
[2025-09-24T09:09:35.437+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:09:35.558+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-24T09:59:33.298+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_d8 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:59:33.306+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_d8 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:59:33.308+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T09:59:33.335+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cleanse_data_d8> on 2025-09-24 08:00:00+00:00
[2025-09-24T09:59:33.341+0000] {standard_task_runner.py:60} INFO - Started process 1023 to run task
[2025-09-24T09:59:33.346+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'cleanse_data_d8', 'scheduled__2025-09-24T08:00:00+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmpshkwzouy']
[2025-09-24T09:59:33.355+0000] {standard_task_runner.py:88} INFO - Job 303: Subtask cleanse_data_d8
[2025-09-24T09:59:33.435+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_d8 scheduled__2025-09-24T08:00:00+00:00 [running]> on host 5d539c157d54
[2025-09-24T09:59:33.539+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='cleanse_data_d8' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-24T08:00:00+00:00'
[2025-09-24T09:59:33.764+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:59:33.777+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=cleanse_data_d8, execution_date=20250924T080000, start_date=20250924T095933, end_date=20250924T095933
[2025-09-24T09:59:33.808+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:59:33.859+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
